opendevin/runtime/e2b/runtime.py

```diff
@@ -20,11 +21,12 @@
 class E2BRuntime(ServerRuntime):
     def __init__(
         self,
+        sandbox_config: SandboxConfig,
```

Comment:
Why not make it as a global config?

Reply:
And should we spend time to refactor e2b, seems not used currently?

Reply:
This is actually part of @neubig's effort in removing global vars: #2760